src/main/webapp/WEB-INF/tags/staticPage.tag

```diff
@@ -40,6 +40,16 @@
               <li class="${currentPage == 'about' ? 'active' : ''}"><a href="about.jsp">About Us</a></li>
               <li class="${currentPage == 'contact' ? 'active' : ''}"><a href="contact.jsp">Contact</a></li>
               <li class="${currentPage == 'terms' ? 'active' : ''}"><a href="terms.jsp">Terms of Use</a></li>
+              <li class="${currentPage == 'requestAccessLinksResend' ? 'active' : ''}">
+                <div class="dropdown-submenu pull-left-and-down">
+                  <button class="dropdown-button">Request
+                    <i class="fa fa-caret-down"></i>
+                  </button>
+                  <div class="dropdown-menu">
+                    <a id="requestAccessLinksResend" href="../../../requestResendLinks.jsp">Resend Access Links</a>
```

Comment:
Link to the short URL. The relative link can begin with `/` to avoid all the `../`